.github/workflows/frontend.yml

```diff
@@ -88,9 +88,14 @@ jobs:
       uses: ./.github/actions/prepare-backend
       with:
         m2-cache-key: 'cljs'
-    - run: yarn run test-unit --coverage --silent
-      name: Run frontend unit tests
+    - name: Run frontend unit tests and collect the code coverage information
+      if: github.ref == 'refs/heads/master'
+      run: yarn run test-unit --coverage --silent
+    - name: Run frontend unit tests
+      if: github.ref != 'refs/heads/master'
+      run: yarn run test-unit
     - name: Upload coverage to codecov.io
+      if: github.ref == 'refs/heads/master'
```

Comment:
nit: `ref_name` is shorter, it's just `master`

Reply:
Can you please update it?
I'm currently AFK.

Reply:
my bad, I was afk as well :(